src/backend/mailservers.nim

```diff
@@ -32,4 +32,12 @@ proc fillGaps*(chatId: string, messageIds: seq[string]): RpcResponse[JsonNode] {
 
 proc requestAllHistoricMessagesWithRetries*(forceFetchingBackup: bool): RpcResponse[JsonNode] {.raises: [Exception].} =
   let payload = %*[forceFetchingBackup]
-  result = core.callPrivateRPC("requestAllHistoricMessagesWithRetries".prefix, payload)
\ No newline at end of file
+  result = core.callPrivateRPC("requestAllHistoricMessagesWithRetries".prefix, payload)
+
+proc requestMoreMessages*(chatId: string): RpcResponse[JsonNode] {.raises: [Exception].} =
+  info "fetching messages"
```

Comment:
I'd put this logging as `debug` instead of `info`, it doesn't seem very important.
Same for line 43.

Reply:
removed, thank you